src/commands/mod.rs

```diff
@@ -2005,14 +2008,71 @@ fn cmd_new(ui: &mut Ui, command: &CommandHelper, args: &NewArgs) -> Result<(), C
         "expected a non-empty list from clap"
     );
     let commits = resolve_base_revs(&workspace_command, &args.revisions)?;
-    let parent_ids = commits.iter().map(|c| c.id().clone()).collect();
+    let parent_ids = commits.iter().map(|c| c.id().clone()).collect::<Vec<_>>();
```

Comment:
Nit: throughout, you can use `.collect_vec()` instead of `.collect::<Vec<_>>`. I think it's from `itertools`.

Reply:
I hadn't noticed the itertools dependency. Done.